buildSrc/src/main/kotlin/com/charleskorn/kaml/build/ConfigureAssemble.kt

```diff
@@ -33,7 +33,8 @@ fun Project.configureAssemble() {
         project.extensions.getByType<PublishingExtension>().publications.names
             .filter { it != "kotlinMultiplatform" }
             .forEach { publicationName ->
-                from(tasks.named("${publicationName}Jar"))
+                // only Jvm and JS targets have jar task. All others have *binaries task
+                tasks.findByName("${publicationName}Jar")?.let { from(it) }
```

Comment:
This will eagerly create the `...Jar` task. Is there another way to achieve the same thing without eagerly creating these tasks? https://docs.gradle.org/current/userguide/task_configuration_avoidance.html explains this further.

Reply:
I didn't find an equivalent in Lazy task api. But I think we can do it another way:
```kotlin
if (publicationName in PUBLICATIONS_WITH_JAR) {
  from(tasks.named(...))
}
```